core/src/main/java/org/frankframework/align/content/JsonDocumentContainer.java

```diff
@@ -15,46 +15,76 @@
 */
 package org.frankframework.align.content;
 
+import java.util.ArrayDeque;
+import java.util.Deque;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
 
 import jakarta.json.stream.JsonGenerator;
-import lombok.Getter;
+
 import org.apache.commons.lang3.NotImplementedException;
-import org.apache.logging.log4j.Logger;
+import org.apache.xerces.xs.XSSimpleTypeDefinition;
 import org.apache.xerces.xs.XSTypeDefinition;
-import org.frankframework.util.LogUtil;
+
+import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
 
 /**
  * Helper class to construct JSON from XML events.
  *
  * @author Gerrit van Brakel
  */
-public class JsonDocumentContainer extends TreeContentContainer<JsonElementContainer>{
-	protected Logger log = LogUtil.getLogger(this.getClass());
+@Log4j2
+public class JsonDocumentContainer {
 
-	private @Getter final String name;
-	private final boolean skipArrayElementContainers;
-	private final boolean skipRootElement;
 	private static final String ATTRIBUTE_PREFIX = "@";
 	private static final String MIXED_CONTENT_LABEL = "#text";
 
 	private static final char[] INDENTOR = "\n                                                                                         ".toCharArray();
```

Comment:
Dit is toch geen indentor, of zie ik het verkeerd

Reply:
Daar heb ik niet over nagedacht... Dat bestond al. Hoe het in de code wordt gebruikt, daar heb ik niet naar gekeken.
Misschien dat een substring hiervan wordt gebruikt afhankelijk van indentation-level?